### PR TITLE
Update ariang from 1.3.2 to 1.3.6

### PR DESCRIPTION
**Pull request by ArielHeleneto**
Maintainer: me / @\<github-user> (find it by checking history of the package Makefile)
Compile tested: (put here arch, model, OpenWrt version)
Run tested: (put here arch, model, OpenWrt version, tests done)

Description:

### CODE REVIEW

File: net/ariang/Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ariang
-PKG_VERSION:=1.3.2
-PKG_RELEASE:=1
+PKG_VERSION:=1.3.6
+PKG_RELEASE:=2
```
Comment (1715173329) [inline net/ariang/Makefile:5]:
```suggestion
PKG_RELEASE:=1
```

This should be reset to `1`, see:
https://openwrt.org/docs/guide-developer/package-policies#versioning

**Review by 1715173329 — COMMENTED** [review]
Commit title should be `ariang: Update to 1.3.6`
And it would be nice to include Changelog in commit message.

Cc maintainer @Ansuel

File: net/ariang/Makefile (hunk 1 above)
Comment (ArielHeleneto) [reply net/ariang/Makefile:5]: fixing

Comment (ArielHeleneto) [reply net/ariang/Makefile:5]: @1715173329 something went wrong when sysncing from openwrt and I need to open an new pull #22125 22125

**Pull request closed without merging**